src/cosmosdb-preview/azext_cosmosdb_preview/_help.py

```diff
@@ -7,6 +7,150 @@
+        --resource-group MyResourceGroup \\
+        --rule-name MyRule \\
+        --start_ip_address MyStartIpAddress \\
+        --end_ip_address MyEndIpAddress \\
+"""
+
+helps['cosmosdb mongocluster firewall rule update'] = """
+type: command
+short-summary: Create a CosmosDB for MongoDB cluster firewall rule.
+examples:
+  - name: Create a CosmosDB for MongoDB cluster firewall rule.
+    text: |
+      az cosmosdb mongocluster firewall rule create \\
+        --cluster-name MyCluster \\
+        --resource-group MyResourceGroup \\
+        --rule-name MyRule \\
+        --start_ip_address MyStartIpAddress \\
+        --end_ip_address MyEndIpAddress \\
+"""
+
+helps['cosmosdb mongocluster firewall rule list'] = """
+type: command
+short-summary: Lists firewall rule on a mongo cluster.
+examples:
+  - name: Lists a Mongo Cluster firewall rule in a resource group.
+    text: |
+      az cosmosdb mongocluster list --cluster-name MyCluster  --resource-group MyResourceGroup
+"""
+
+helps['cosmosdb mongocluster firewall rule show'] = """
+type: command
+short-summary: Get a Mongo Cluster firewall rule.
+examples:
+  - name: Gets a Mongo Cluster firewall rule. If the firewall rule doesnot exist a NotFound response is returned.
+    text: |
+      az cosmosdb mongocluster show --cluster-name MyCluster --resource-group MyResourceGroup --rule-name MyRuleName
+"""
+
+helps['cosmosdb mongocluster firewall rule delete'] = """
+type: command
+short-summary: Delete a Mongo Cluster firewall rule.
+examples:
+  - name: Delete a Mongo Cluster firewall rule. If the firewall rule doesnot exist a NotFound response is returned.
+    text: |
+      az cosmosdb mongocluster show --cluster-name MyCluster --resource-group MyResourceGroup --rule-name MyRuleName
+"""
+
+helps['cosmosdb mongocluster create'] = """
+type: command
+short-summary: Create a CosmosDB for MongoDB cluster.
+examples:
+  - name: Create a CosmosDB for MongoDB cluster.
+    text: |
+      az cosmosdb mongocluster create \\
+        --cluster-name MyCluster \\
+        --resource-group MyResourceGroup \\
+        --location MyLocation \\
+        --administrator-login MyAdminUser \\
+        --administrator-login-password MyAdminPassword \\
+        --server-version 5.0 \\
+        --shard-node-sku "M30" \\
+        --shard-node-ha true \\
+        --shard-node-disk-size-gb 128 \\
+        --shard-node-count 2
+  - name: Create a CosmosDB for MongoDB cluster with PITR option.
+    text: |
+      az cosmosdb mongocluster create \\
+        --cluster-name MyCluster \\
+        --resource-group MyResourceGroup \\
+        --location MyLocation \\
+        --administrator-login MyAdminUser \\
+        --administrator-login-password MyAdminPassword \\
+        --server-version 5.0 \\
+        --shard-node-sku "M30" \\
+        --shard-node-ha true \\
+        --shard-node-disk-size-gb 128 \\
+        --shard-node-count 2
+        --restore-point-in-time-utc "2023-01-13T20:07:35Z"
```

Comment:
Open question: Do we want to exclude the restore parameters while the feature is disabled? I assume we can come back to do an additional package release to add the options in CLI when the feature is enabled.

Reply:
I that in the spec, we should keep the API consistent with the spec. Do we silently ignore the parameter or throw error from the service ?

https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2023-03-01-preview/mongoCluster.json#L948

Reply:
We throw an error. Fine to keep this is in so we are aligned with the spec.

Reply:
ok